lib/api.user.js

```diff
@@ -349,6 +349,20 @@ var deleteEmailToken = module.exports.deleteEmailToken = function(restCtx, userI
 };
 
 /**
+ * Get a user's most recent groups through the REST API.
+ *
+ * @param  {RestContext}     restCtx             Standard REST Context object that contains the current tenant URL and the current user credentials
+ * @param  {String}          userId              User id whose recent groups you wish to receive
+ * @param  {Function}        callback            Standard callback function
+ * @param  {Object}          callback.err        Standard error object, if any
+ * @param  {Group[]}         callback.response   The user's most recently visited groups
+ */
+var getRecentlyVisitedGroups = module.exports.getRecentlyVisitedGroups = function(restCtx, userId, callback) {
+    var url = '/api/user/' + RestUtil.encodeURIComponent(userId) + '/recent';
```

Comment:
isn't `/recent` a little vague semantically? we might want to add other "recent" resources in the future.